### PR TITLE
Update calc.sh

### PR DESCRIPTION
**Pull request by appars**
Hi, I have fixed bug01 subtraction issue. Can you please review?

### CODE REVIEW

File: calc.sh [hunk 1]
```diff
@@ -13,10 +13,10 @@ echo "Enter your choice"
 read ch
 
 case $ch in
-1)sum=`expr $n1 + $n2`
+1)sum=`expr $n1 - $n2`
```
Comment (Lakshmikanthwodeyar) [inline calc.sh]: @appars Why adding having -, it should + rt?

Comment (Bukter) [inline calc.sh]: @appars 
In addition sign has to  be plus

File: calc.sh [hunk 2]
```diff
@@ -13,10 +13,10 @@ echo "Enter your choice"
...
    echo "Sum ="$sum;;
 
-2)sum=`expr $n1 + $n2`
+2)sum=`expr $n1 - $n2`
```
Comment (Lakshmikanthwodeyar) [inline calc.sh:19]: Can you please add a comment like bug fixng

**Review by Lakshmikanthwodeyar — CHANGES_REQUESTED** [review]
please check my review comments?

File: calc.sh (hunk 2 above)
Comment (Bukter) [inline calc.sh:19]: Can you pls give a comment inwhich line you are fixing bug

**Review by Bukter — CHANGES_REQUESTED** [review]
pls look in tomy comm

File: calc.sh (hunk 2 above)
Comment (appars) [reply calc.sh:19]: Thanks for the review comments. I have corrected it. Please review and approve.

Comment (appars) [reply calc.sh:19]: Added a comments as well.

**Comment by appars** [comment]
@Bukter @Lakshmikanthwodeyar I have implemented all the review comments. Can you please review and approve the same.

**Review by Lakshmikanthwodeyar — APPROVED** [review]
LGFM

**Review by Bukter — APPROVED** [review]
looks fin

**Pull request merged**